auto_round/data_type/nvfp.py

```diff
@@ -75,9 +75,9 @@ def nv_fp4(tensor, bits=4, group_size=16, v=0, **kwargs):
     tensor, orig_shape, pad_len = reshape_pad_tensor_by_group_size(tensor, group_size)
     tensor_amax = tensor.abs().max().to(torch.float32)
     global_scale = FLOAT8_E4M3_MAX * FLOAT4_E2M1_MAX * get_reciprocal(tensor_amax)
-    qdq_res, output_scale = ref_nvfp4_quant(tensor, global_scale, group_size, v)
+    qdq_res, scale = ref_nvfp4_quant(tensor, global_scale, group_size, v)
     qdq_res = revert_tensor_by_pad(qdq_res, orig_shape=orig_shape, pad_len=pad_len)
-    return qdq_res.to(orig_dtype), output_scale, None
+    return qdq_res.to(orig_dtype), scale, global_scale, None
```

Comment:
{"scale":scale,"global_scale":global_scale}, then the number of return elements could align with other formats